halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java

```diff
@@ -53,6 +53,7 @@
 
   Boolean serviceAccount;
   int cacheThreads = 1;
+  Long cacheIntervalSeconds = 30L;
```

Comment:
Removing the default means removing it from here too, yes?

Reply:
Missed that, but got to say that there are changes that already went in with the default values being set in multiple places, may be a cleanup is required.

Reply:
Can you show me where?

Reply:
The line just above this one https://github.com/spinnaker/halyard/blob/master/halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java#L55 ,  https://github.com/spinnaker/halyard/blob/master/halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/kubernetes/KubernetesAddAccountCommand.java#L121, Basically this variable cacheThreads

Reply:
Aaah ok...some other field, but not cacheIntervalSeconds.  I'm happy to leave that cleanup for another PR/time.